import mysql.connector
user = 'root'
password = ''
host = 'localhost'
database = 'inventario'
try:
    conexion = mysql.connector.connect(
        host=host,
        user=user,
        passwd = "",
        database=database,
    )
    print("Conexi贸n exitosa")
except mysql.connector.Error as error:
    print("Error de conexi贸n:", error)
finally:
    # Si la conexi贸n se estableci贸, la cerramos
    if 'conexion' in locals() or 'conexion' in globals():
        conexion.close()